lib/disconnected_pulp.rb

```diff
@@ -332,13 +345,17 @@ def export(target_basedir = nil, repoids = nil, overwrite = false, onlycreate =
       cmd = "tar -C /var/www/pulp_puppet/http/repos -c -h -z -p -f - . |  split -d -b 4600M - #{target_basedir}/content-export-puppet-"
       system(cmd)
     end
+    if File.exist?("/var/lib/pulp/published/http/isos")
+      cmd = "tar -C /var/lib/pulp/published/http/isos -c -h -z -p -f - . |  split -d -b 4600M - #{target_basedir}/content-export-isos-"
+      system(cmd)
+    end
     # Write out simple script to expand split up archives
-    unsplit_script = "#!/bin/bash\n\n"\
-                     "cat content-export-* | tar xzpf -\n\n"\
-                     "echo \"*** Done expanding archives. ***\"\n"
-    # puts unsplit_script
     f = File.open("#{target_basedir}/expand_export.sh", 'w')
-    f.write(unsplit_script)
+    f.write("#!/bin/bash\n\n")
+    f.write("cat content-export-* | tar xzpf -\n\n")
+    f.write("cat content-export-puppet-* | tar xzpf -\n\n") if File.exist?("/var/www/pulp_puppet/http/repos")
```

Comment:
I used the same directory that was being used on line 344 https://github.com/fusor/katello-utils/blob/cfme-patch/lib/disconnected_pulp.rb#L344

I just assumed because we're exporting to a directory that isn't being tarred. If it is indeed /var/lib/pulp/.... then we can remove it and probably should fix line 344.